### PR TITLE
chore: switching from CLA to DCO

### LINKED ISSUE #1158: Update repo from CLA to DCO
Update repo from CLA to DCO

### PR DESCRIPTION
**Pull request by prsdthkr**
### Description
Migrate Fundamental React from CLA to DCO

The SAP Open Source Program Office is working on adopting external standards and best practices wherever possible and making contributions to our open source projects as easy as possible. Therefore, some adjustments are required in this repository:
* Switch from Contributor License Agreement (CLA) to Developer Certificate of Origin (DCO)
* Generate license and copyright information with the 'Reuse' tool of the Free Software Foundation Europe (FSFE)
* Adjust the license file to use standard text for [Apache License, Version 2.0
](https://opensource.org/licenses/Apache-2.0)

fixes #1158

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Fundamental React
 
+[![REUSE status](https://api.reuse.software/badge/github.com/SAP/fundamental-react)](https://api.reuse.software/info/github.com/SAP/fundamental-react)
```
Comment (prsdthkr) [inline README.md:3]: @droshev fundamental@sap.com shoudl have recieved an email to register our project to use this badge.

Comment (jbadan) [reply README.md:3]: https://api.reuse.software/info/github.com/SAP/fundamental-react We failed lol

**Review by jbadan — APPROVED** [review]
LGTM I assume by adding the License and .reuse file it will pass our reuse scan once this is merged to master?

**Comment by prsdthkr** [comment]
> LGTM I assume by adding the License and .reuse file it will pass our reuse scan once this is merged to master?

Yep lets hope. 🤞

**Pull request merged**